frontend/Dockerfile

```diff
@@ -38,3 +38,7 @@ EXPOSE 3000
 RUN npm run build
 ENV API_SERVER_ADDRESS http://localhost:3001
 CMD node dist/server.js ../client/ 3000
+
+# HACK: curl is required for running liveness/readiness probe inside the container
```

Comment:
JFYU: alpine has `wget`

Reply:
Ohh, thanks! I will switch to wget then.